shell/src/main/java/org/apache/accumulo/shell/commands/MergeCommand.java

```diff
@@ -53,13 +53,9 @@ public int execute(final String fullCommand, final CommandLine cl, final Shell s
       size = ConfigurationTypeHelper.getFixedMemoryAsBytes(cl.getOptionValue(sizeOpt.getOpt()));
     }
     if (startRow == null && endRow == null && size < 0 && !all) {
-      shellState.getWriter().flush();
-      String line = shellState.getReader()
-          .readLine("Merge the entire table { " + tableName + " } into one tablet (yes|no)? ");
-      if (line == null) {
-        return 0;
-      }
-      if (!line.equalsIgnoreCase("y") && !line.equalsIgnoreCase("yes")) {
+      if (!shellState
```

Comment:
Do we not want `force` to take affect here for this command? It would be a change in behavior if added, but just curious.